hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/StoreContext.java

```diff
@@ -25,6 +25,7 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
 
+import org.apache.hadoop.fs.s3a.S3ObjectStorageClassFilter;
```

Comment:
can you move down to the rest of the org.apache. 
these guava things are in the wrong block due to the big search and replace which created them